### DISCUSSION TITLE
Specific container hardware requirements

### DISCUSSION THREAD

**Discussion by saendu** [langgenius/dify#7814] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I am using the terra form script mentioned in the docs (https://github.com/nikawang/dify-azure-terraform) to run Dify in my Azure subscription. 
I want to safe as much costs as possible. Currently we are not using the environment for production, so we don't have a lot of traffic. 

Can anybody give me more detailed information about the minimum requirements on each container? 
Currently I am using the following allocations:

```
1.	nginx
	•	cpu: 0.5
	•	memory: “1Gi”
2.	ssrfproxy
	•	cpu: 0.5
	•	memory: “1Gi”
3.	sandbox
	•	cpu: 0.5
	•	memory: “1Gi”
4.	api
	•	cpu: 2
	•	memory: “4Gi”
5.	web
	•	cpu: 2
	•	memory: “4Gi”
```

The postgres database runs on `P4 (120 iops)` (smalles available). 

The only recommondation that I've found in the docs is this very generic one regarding docker:

> Before installing Dify, make sure your machine meets the following minimum system requirements:
> CPU >= 2 Core
> RAM >= 4GB

**Comment by Eugene-Zh** [comment]
Yes，there is a minimum system requirements in README file. Do we have a recommend requirements for the best experience besides?
